packages/react/__tests__/__snapshots__/PublicAPI-test.js.snap

```diff
@@ -4303,16 +4303,6 @@ Map {
     },
     "render": [Function],
   },
-  "ListItem" => Object {
-    "propTypes": Object {
-      "children": Object {
-        "type": "node",
-      },
-      "className": Object {
-        "type": "string",
-      },
-    },
-  },
```

Comment:
This shouldn't be changing. Might need to revert the change to how the component is exported.